### ISSUE TITLE
Support for pressure considered?

### ISSUE THREAD

**Issue by anilgulecha** [embiem/react-canvas-draw#125]
Pls consider adding pressure based thickness support. Touch events can indicate the pressure on the drawing (which can be used to set the size of point/line/curve drawn).

Eg:
https://patrickhlauke.github.io/touch/tracker/tracker-force-pressure.html
https://viliusle.github.io/miniPaint/

This would make the component quite versatile.

**Comment by sifatdipta** [comment]
This honestly would make this program perfect! I actually came here to suggest this. 

Would be awesome!

**Comment by embiem** [comment]
Should be a low-hanging fruit to add using https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pressure, right?

**Comment by sifatdipta** [comment]
> Should be a low-hanging fruit to add using https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pressure, right?

Yeah I believe so, you can also look into https://github.com/stuyam/pressure

**Comment by sifatdipta** [comment]
Would adding a eraser mode be too difficult? I was playing with your codes earlier to come up with a solution for both pressure and eraser mode.
